src/Sylius/Bundle/CoreBundle/Resources/config/validation/ChannelPricing.xml

```diff
@@ -27,20 +27,35 @@
                 <option name="minMessage">sylius.channel_pricing.price.min</option>
                 <option name="groups">sylius</option>
             </constraint>
+            <constraint name="LessThan">
```

Comment:
The `Range` constraint has a `max` option, maybe we should use it. Or use the `Type` constraint, which as far as I know, checks the range of an integer.